docs/connectors.rst

```diff
@@ -28,7 +33,6 @@ If you want to connect to facebook using the run script, e.g. using
 you need to supply a ``fb_credentials.yml`` with the following content:
 
 .. literalinclude:: ../examples/moodbot/fb_credentials.yml
-   :linenos:
```

Comment:
originally took them out because were breaking the theme - that's now fixed. In general we should have them when we actually want to refer to specific lines, but then the line highlights are also a good tool for that (see `slotfilling.rst`)